contracts/BuyBackBurner.sol

```diff
@@ -102,17 +102,14 @@ contract BuyBackBurner {
     }
 
     function checkPoolPrices(
-        address nativeToken,
-        address memeToken,
+        address token0,
+        address token1,
         address uniV3PositionManager,
-        uint24 fee,
-        bool isNativeFirst
+        uint24 fee
     ) external {
```

Comment:
This contract should be as generic as possible so we can reuse it elsewhere, moved the sorting out